packages/transaction/src/tx-builder.ts

```diff
@@ -267,7 +267,7 @@ export class TxBuilder {
     }
 
     // Spend ETH containing notes until it hits the number
-    spendables.sort((a, b) => (a.eth().gt(b.eth()) ? -1 : 1))
+    spendables.sort((a, b) => (a.eth().gt(b.eth()) ? 1 : -1))
```

Comment:
So it uses large ETH notes first now? This was trying to consume small penny ETH notes as possible. Was there any problem w/ it?

Reply:
Ah yes, I forgot to document this. If the user has (for example) 5 ether total, but 4 notes with 0.01 Eth, then the max spend is 0.04, which is unexpected. I changed this so testing would be easier.

We should refactor this to use the smallest notes possible while still spending the full amount. This will take a bigger refactor/change. I documented this in #284

Reply:
thank u @JChanceHud !

Reply:
looks good to me!